doc/md_src_files/060_decision-trees.md

```diff
@@ -101,13 +101,54 @@ Once the decision points are selected and the prioritization labels agreed upon,
 Making the decision process accessible has a lot of benefits.
 Unfortunately, it also makes it a bit too easy to overcomplicate the decision.
 
+The academic literature surrounding decision tree quality measurement is primarily concerned with measuring classification errors given a particular tree and a labeled data set.
+In our case, we are not attempting to fit a tree to data.
+Rather, we are interested in producing usable trees that minimize extraneous effort.
+To that end, a brief examination of what qualities decision tree construction  methods optimize for is warranted.
+
+Decision tree construction methods must address four significant concerns: feature selection, feature type, overfitting, and parsimony.
+
+Feature selection is perhaps the most important consideration for SSVC, because it directly affects the information gathering requirements placed on the analyst attempting to use the tree.
+Each decision point in SSVC is a feature.
+
 The SSVC version 1  ~applier~ deployer tree had 225 rows when we wrote it out in long text form.
 It only has four outcomes to differentiate between.
 Thus, on average that decision process treats one situation (combination of decision values) as equivalent to 65 other situations.
 If nothing else, this means analysts are spending time gathering evidence to make fine distinctions that are not used in the final decision.
 The added details also make it harder for the decision process to accurately manage the risks in question.
 This difficulty arises because more variance and complexity there is in the decision increases the possibility of errors in the decision process itself.
 
+Different feature types (nominal, ordinal, interval, ratio) require slightly different approaches when building decision trees.
```

Comment:
I think it's enough to say that the metrics being proposed only work on ordinal decision values. The types of measurement stuff is explained in `# Representing Information for Decisions About Vulnerabilities`.

Reply:
adjusted in 6fbb0de